src/components/AnimalDetail/AnimalDetail.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+// styles 
+import './AnimalDetail.css'
+
+const AnimalDetail = ({ image, name, latin, decsription, home, biotop, food, size, zoos, listOfZoos }) => {
+
+    return (
+        <>
+            {listOfZoos && (
+                <div className="detail">
+                    <div className="detail__content">
+                        <div className="detail__header">
+                            <img className="detail__image" src={image} alt={name} />
+                            <div className="detail__title">
+                                <h2 className="detail__name"><span>{name}</span></h2>
+                                <div className="detail__latin"><span>{latin}</span></div>
+                            </div>
+                        </div>
+                        <div className="detail__info">
+                            <p className="detail__desc">{decsription}</p>
+                            <div className="detail__items">
+                                <div className="detail__item">
+                                    <h3>Domovina</h3>
+                                    <p>{home}</p>
+                                </div>
+                                <div className="detail__item">
+                                    <h3>Biotop</h3>
+                                    <p>{biotop}</p>
+                                </div>
+                                <div className="detail__item">
+                                    <h3>Potrava</h3>
+                                    <p>{food}</p>
+                                </div>
+                                <div className="detail__item">
+                                    <h3>Velikost</h3>
+                                    <p>{size}</p>
+                                </div>
+                                <div className="detail__zoo">
+                                    <h3>Najdete v těchto zoo</h3>
+                                    <p>{zoos.map(
+                                        (zoo) => {
+                                            const zooName = listOfZoos.find(el => el.id === zoo)
+                                            return (
+                                                zoo === 7 ?
```

Comment:
jak jsem říkal na slacku, tohle.... funguje jen ve specifickém případě. Funguje to jen když:
- víš kolik je celkem zoo a znáš ID poslední zoo
- pole zoo ve zvířatech je seřazené
- a všechna zvířata jsou v zoo 7. Co když nějaké zvíře nebude v zoo 7?

Logicky na to jdeš ale správně, chceš aby čárka nebyla za posledním. Základní algoritmus/postup je 
```
zoos.map((zoo, index) => {
  const zooName = listOfZoos.find(el => el.id === zoo);
  return index + 1 === zoos.length ? (
    <span key={zoo}>{zooName.jmeno.substring(4)}</span>
  ) : (
    <span key={zoo}>{zooName.jmeno.substring(4)}, </span>
  );
});
```
Takhle napíšeš podmínku, která fakt kontroluje že se jedná v poli o poslední prvek

Reply:
Kdybys chtěla být fancy, tak uděláš:
```
zoos
  .map((zoo, index) => {
    const zooName = listOfZoos.find(el => el.id === zoo);
    return <span key={zoo}>{zooName.jmeno.substring(4)}</span>;
  })
  .join(', ');
```